processor/src/decoder/trace.rs

```diff
@@ -17,6 +17,10 @@ pub const USER_OP_HELPERS: Range<usize> = Range {
     end: NUM_HASHER_COLUMNS,
 };
 
+/// Total number of user helper registers in the decoder trace.
+#[allow(dead_code)]
+pub const TOTAL_USER_OP_HELPERS: usize = USER_OP_HELPERS.end - USER_OP_HELPERS.start;
```

Comment:
We should probably use `#[cfg(test)]` attribute here instead of `#[allow(dead_code)]`.

Also nit: I'd probably name this `NUM_USER_OP_HELPERS`.

Reply:
I've updated the attribute to `#[cfg(test)]`, thanks!  I also use `NUM_USER_OP_HELPERS` from `core/decoder` instead of defining my own here.